.github/workflows/windows_tests.yml

```diff
@@ -29,10 +29,10 @@ jobs:
 
       - name: download and install zig
         run: |
-          curl.exe -f --output "C:\zig-windows-x86_64-0.9.1.zip" --url https://ziglang.org/download/0.9.1/zig-windows-x86_64-0.9.1.zip
+          curl.exe -f --output "C:\zig-windows-x86_64-0.11.0.zip" --url https://ziglang.org/download/0.11.0/zig-windows-x86-0.11.0.zip
           cd C:\
-          7z x zig-windows-x86_64-0.9.1.zip
-          Add-Content $env:GITHUB_PATH "C:\zig-windows-x86_64-0.9.1\"
+          7z x zig-windows-x86_64-0.11.0.zip
+          Add-Content $env:GITHUB_PATH "C:\zig-windows-x86_64-0.11.0\"

```

Comment:
@Anton-4 would you be able to look at this please? I am not sure why this hasn't worked. I have updated using the link from Zig downloads. I cannot run these commands locally as they must be specific to github's windows machine?

Reply:
The x86 in the download link needed to be x86_64, I've pushed a fix :)